main/guides/getting-started/swapaoo-making-a-payment-explainer.md

```diff
@@ -0,0 +1,63 @@
+# Making Payments in Agoric Smart Contracts
+In this document, we'll explain how to send a payment to someone using their `agoric1...` address from an Agoric smart contract using a deposit facet.
+
+## Using a depositFacet
+Let's take a look at the following code snippet from the Swaparoo contract from [dapp-agoric-basics](https://github.com/Agoric/dapp-agoric-basics):
+```js
+const secondDepositFacet = await E(depositFacetFromAddr).lookup(
+  secondPartyAddress,
+  'depositFacet',
+);
+
+await E(secondDepositFacet).receive(secondSeatInvitation);
+
+return 'invitation sent';
+```
+
+## Step-by-Step Explanation
+### Retrieving the Deposit Facet:
+- `depositFacetFromAddr` is an object that provides a lookup function for deposit facets associated with addresses. The Swaparoo contract is provided with a `depositFacetFromAddr` by the proposal (`swaparoo.proposal.js`) as a parameter in the `startGovernedIntance` call (see below):
+  - ```js
+    const it = await startMyGovernedInstance(
+      {
+        zoe,
+        governedContractInstallation: installationP,
+        label: contractName,
+        terms: {},
+        privateArgs: {
+          storageNode,
+          marshaller,
+          namesByAddressAdmin,
+        },
+      },
+      {
+        governedParams,
+        timer: chainTimerService,
+        contractGovernor,
+        governorTerms: {},
+        committeeCreatorFacet: E.get(committeeKitP).creatorFacet,
+      },
+    );```
+  - An example of an address might be `agoric1ydzxwh6f893jvpaslmaz6l8j2ulup9a7x8qvvq`.
+- The lookup function is called with `secondPartyAddress` and `'depositFacet'` as arguments to retrieve the deposit facet associated with the `secondPartyAddress`.
+- Since `lookup` is an asynchronous function that returns a promise, the `await` keyword is used to wait for the promise to resolve.
+- The resulting deposit facet is stored in the `secondDepositFacet` variable.
+
+### Making the Payment:
+- `secondDepositFacet` represents the deposit facet obtained in the previous step.
+- The `receive` method is called on `secondDepositFacet`, passing `secondSeatInvitation` as an argument.
+- `secondSeatInvitation` is an [Invitation](https://docs.agoric.com/glossary/#invitation) to participate in the second seat (recall that invitations are payments).
```

Comment:
**important**

> use relative links instead of absolute ones for any links to files or folders in the Documentation repo
  -- [Writing Links in CONTRIBUTING](https://github.com/Agoric/documentation/blob/main/CONTRIBUTING.md#writing-links)

Reply:
Links fixed